From 6b680149626a8b8b5524656ab4b2d9a4ce6cfbae Mon Sep 17 00:00:00 2001
From: Pino Toscano <pino@kde.org>
Date: Tue, 7 Oct 2014 07:59:24 +0200
Subject: [PATCH] bump the version to 2.3.2

This way digikam can know this version has the needed bugfixes for it.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a4cbb05..ed57e12 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -78,7 +78,7 @@ MACRO_LOG_FEATURE(EXIV2_FOUND "Exiv2" "Required to build libkexiv2." "http://www
 # Library API version
 SET(KEXIV2_LIB_MAJOR_VERSION "2")
 SET(KEXIV2_LIB_MINOR_VERSION "3")
-SET(KEXIV2_LIB_PATCH_VERSION "1")
+SET(KEXIV2_LIB_PATCH_VERSION "2")
 
 # Suffix to add at end of version string. Usual values are:
 # "-git"   : alpha code unstable from git. Do not use in production
-- 
2.1.1

